src/libraries/System.Private.CoreLib/src/System/Threading/ExecutionContext.cs

```diff
@@ -314,6 +314,30 @@ internal static void RunFromThreadPoolDispatchLoop(Thread threadPoolThread, Exec
             edi?.Throw();
         }
 
+        internal static void RunForThreadPoolUnsafe(ExecutionContext executionContext, ContextCallback callback, object state, Thread threadPoolThread)
+        {
+            Debug.Assert(threadPoolThread == Thread.CurrentThread);
+            // We aren't running in try/catch as if an exception is directly thrown on the ThreadPool either process
+            // will crash or its a ThreadAbortException.
```

Comment:
This comment should be revised now, right?

Reply:
Updated; and previous comment that's already there